css-grid-1/grid-items/grid-item-containing-block-004.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Grid item sizing in a positioned grid container</title>
+<link rel="author" title="Tomek Wytrebowicz" href="mailto:tomalecpub@gmail.com">
+<link rel="help" href="http://www.w3.org/TR/css-grid-1/#grid-items" title="4. Grid Items">
+<meta name="assert" content="A grid item is sized within the containing block defined by its grid area">
+<link rel="match" href="../../css21/reference/ref-filled-green-100px-square.xht">
+<style>
+    #grid {
+        display: grid;
+        position: absolute;
+        height: 200px;
+        width: 200px;
+        grid-template-rows: 1fr 1fr;
+        grid-template-columns: 1fr 1fr;
+    }
+
+    #test-item {
+        background-color: green;
+        width: 100%;
+        height: 100%;
+    }
+</style>
+<body>
+    <p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
```

Comment:
The problem with this test is that you cannot see red anyway. There's no red in the test.
For example, in a browser without grid support you'll see a 200x200 green square.

So please add something like we have in other tests:

            #reference-overlapped-red {
                position: absolute;
                background-color: red;
                width: 100px;
                height: 100px;
                z-index: -1;
            }

    <div id="reference-overlapped-red"></div>